src/components/ShareStructure/ShareStructures.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div id="share-structures">
+
+    <resolution-date-dialog
+      attach="#share-structures"
+      :dialog="toggleResolutionDateDialog"
+      @emitClose="toggleResolutionDateDialog = false"
+    />
+
+    <share-structure
+      :is-edit-mode="isEditMode"
+      :isCorrection="isCorrectionView()"
```

Comment:
Maybe we should make this a getter eh? (Not sure if that will work as it's declared in a mixin, but if it does work then it will be consistent and maybe a tad faster due to getters being cached.)

Reply:
That was definitely the first choice, is there a clean way to get the route within the vuex getters? 
Time for some research! 😁

Reply:
I would only have getters in the Vuex code that do something with `state`.

I'm wondering if we need a Getter Mixin or something for non-state getters? Or leave those in Common Mixin?

Reply:
There is a way to apply global mixins to the store i believe, 
`new Vuex.Store<any>({ mixins: { ...MixinHere } })`
I'm sure there is something more too it, but it's possible!

Reply:
Good find. Is this something you want to champion in this project (and possibly NR2 later)? I leave it with you...